flink-streaming-java/src/test/java/org/apache/flink/streaming/runtime/io/benchmark/StreamNetworkBenchmarkEnvironment.java

```diff
@@ -213,7 +213,6 @@ protected ResultPartitionWriter createResultPartition(
 			int channels) throws Exception {
 
 		ResultPartition resultPartition = new ResultPartitionBuilder()
-			.setJobId(jobId)
```

Comment:
We should actually also wrap `resultPartition` here to take wrapping into account in benchmark tests. Probably with `NoOpTaskActions`, at least forcing the wrapper to be called.